File: kraken/src/main/java/glitch/kraken/object/json/list/FeatureStreams.java
package glitch.kraken.object.json.list;

import com.google.common.collect.ImmutableList;
import glitch.api.objects.json.interfaces.OrdinalList;
import glitch.kraken.object.json.FeatureStream;
import lombok.Data;

@Data
public class FeatureStreams implements OrdinalList<FeatureStream> {
    private final ImmutableList<FeatureStream> data;
}

File: kraken/src/main/kotlin/glitch/kraken/object/json/collections/FeatureStreams.kt
package glitch.kraken.`object`.json.collections

import glitch.api.objects.json.interfaces.OrdinalList
import glitch.kraken.`object`.json.FeatureStream

data class FeatureStreams(
        override val data: List<FeatureStream>
) : OrdinalList<FeatureStream>
